Templates/CRMs/Odoo/Generic JSON order file to Odoo/long-description.md

```diff
@@ -0,0 +1,22 @@
+This template will retrieve an order file from an SFTP server. It will perform the necessary mapping to Odoo order format, and transfer the mapped orders to Odoo.
+Using this template, you can insert new orders to Odoo.
+
+![Template](assets/Generic_JSON_order_file_to_Odoo.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- SFTP address, username and password are required to access the file.
+- Odoo user exists for performing the synchronization.
+- Odoo has a database with usable products, partners, addresses and a pricelist for the synchronization.
+
+# Implementation and Usage Notes
+
+This template is used to insert order data to Odoo using Odoo models for order and order line. Other properties, such as activation, deactivation and deletion is not implemented in this template.
+Process variables include fields for both SFTP and Odoo credentials, SFTP and Odoo addresses, SFTP file path, SFTP file name and Odoo database name.
```

Comment:
Please include a JSON file example that this template is able to handle.

Reply:
Example data added!